.github/workflows/dart_ci.yml

```diff
@@ -46,9 +43,8 @@ jobs:
         run: |
           # Analyze lib to ensure public APIs don't depend on build-to-cache files,
           # which could cause analysis issues for consumers who haven't run a build yet.
-          dartanalyzer lib
```

Comment:
Had to switch over to `dart analyze` since `dartanalyzer` isn't available in the latest stable SDK. 

I believe this should be equivalent since we haven't run a build yet.